libs/SalesforceSDK/res/drawable/sf__fix_status_bar.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:id="@+id/sf__status_bar_background" android:height="100dp" >
+        <shape android:shape="rectangle">
+            <size android:height="100dp" android:width="100dp"/>
+            <solid android:color="@color/sf__primary_color"/>
+        </shape>
+    </item>
+
+    <item android:id="@+id/sf__nav_bar_background" android:height="100dp">
+        <shape android:shape="rectangle">
+            <size android:height="100dp" android:width="100dp"/>
+            <solid android:color="@android:color/transparent"/>
+        </shape>
+    </item>
+</layer-list>
```

Comment:
This is sort of a hack to fill in the status and navigation bars with the appropriate color since they are now forced transparent and completely unsettable in Edge to Edge.  Note that I am changing the height of the top rectangle to match the device status bar height in code!